resource/research2022-test/session_map.json

```diff
@@ -0,0 +1,14 @@
+{
+  "SessionSetMap": [
```

Comment:
@nelson-w I have made the assumption that test files should have the same data as prod files... is this assumption incorrect?

Reply:
Yes